clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1839,4 +1845,30 @@ private FetchResponse fetchResponse(TopicPartition partition, long fetchOffset,
             this.count = count;
         }
     }
+
+    private java.util.function.Consumer<Duration> spyClose = null;
```

Comment:
@chia7712 We normally declare fields at the start of the class rather than the middle. And this is accessed in common code in the class as well. Looking at the test itself, perhaps it would be better to add a test with mocking that is self-contained rather than doing it this way. Perhaps something like:
```
KafkaConsumer consumer = EasyMock.partialMockBuilder(KafkaConsumer.class)
        .addMockedMethod("close", Duration.class).createMock();
consumer.close(Duration.ofSeconds(1));
EasyMock.expectLastCall();
EasyMock.replay(consumer);
consumer.close(1, TimeUnit.SECONDS);
EasyMock.verify(consumer);
```
What do you think?

Reply:
Thanks for the beautiful test case. Will update the PR asap.